packages/calypso-e2e/src/hooks/recording.ts

```diff
@@ -0,0 +1,111 @@
+/**
+ * External dependencies
+ */
+import { unlink, renameSync } from 'fs';
+import { Context } from 'mocha';
+
+/**
+ * Internal dependencies
+ */
+import { getTargetLocale, getTargetScreenSize } from '../browser-manager';
+import { getDateString, getVideoDir } from '../media-helper';
+
+export async function clearFailedTest( this: Context ): Promise< void > {
+	// String here would be more logical as combination of magellan + mocha should result
+	// in at most one failure per run of mocha.
+	// However, using a string here causes the recorded failures to not survive the context
+	// when the after hook is run whereas an array would.
```

Comment:
Why an array works?

Reply:
I am not sure. I tried multiple times and could not pinpoint why a `string` does not and `array` preserves data.